Tests/UITests/TestApp/ContentView.swift

```diff
@@ -0,0 +1,34 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2022 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import FHIRQuestionnaires
+import SpeziQuestionnaire
+import SwiftUI
+
+
+struct ContentView: View {
+    @EnvironmentObject var standard: ExampleStandard
+    @State var surveyResponseCount = 0
+    @State var displayQuestionnaire = false
+    
+    
+    var body: some View {
+        Text("No. of surveys complete: \(surveyResponseCount)")
+            .onReceive(standard.objectWillChange) { _ in
+                Task { @MainActor in
+                    surveyResponseCount = await standard.surveyResponseCount
+                }
+            }
```

Comment:
Now that you have enforced that `surveyResponseCount` can only be accessed from the `@MainActor`, Swift is actually smart enough to see that this closure will also always be executed in the main actor (enforced by SwiftUI) and you can therefore access the property without the need for an `await`:
```suggestion
        Text("No. of surveys complete: \(standard.surveyResponseCount)")
```